doc/en/user/source/community/schemaless-features/install.rst

```diff
@@ -0,0 +1,13 @@
+Installing the Schemaless Features module
+=========================================
+
+#. Download :download_community:`schemaless-features-plugin` nightly GeoServer community module `builds <https://build.geoserver.org/geoserver/main/community-latest/>`__).
+   
+   .. warning:: Verify that the version number in the filename corresponds to the version of GeoServer you are running (for example geoserver-|release|-schemaless-features-plugin.zip above).
+
+#. Extract the contents of the archive into the ``WEB-INF/lib`` directory of the GeoServer installation.
+
+#. On restart the ``MongoDB Schemaless`` vector source option will be available from the ``New Data Source`` page:
+
+.. figure:: images/new-data-sources-schemaless.png
```

Comment:
This looks like it won't match the next file (which I assume is the screenshot you wanted). The directory and filename are both different.

Reply:
There are two images folder, one in the schemaless-features and another one in the mongo folder. This one is actually matching the right png. Anyway there was a missed png in the index.rst of the mongo folder. Fixed